templates_old/index.html

```diff
@@ -0,0 +1,49 @@
+{%include templates/header.html%}
+<div class="hero">
+    <div class="hero-inner">
+        <h1>facebooc</h1>
+        <h2>
+            The best social network you've never heard of!
+        </h2>
+        <div class="btn-group">
+            <a class="btn" href="/signup/"role="button">
+                REGISTER NOW
+            </a>
+        </div>
+        <p>
+            Or <a href="/login/">LOGIN</a> if you heard about Facebooc before it was cool.
+        </p>
+    </div>
+</div>
+
+<div class="content-wrapper">
+    <div class="section-about">
+        <h3>What is facebooc?</h3>
+        <p>
+            A proof-of-concept Facebook clone written in C. The only dependency is SQLite3.
```

Comment:
We might remove "proof-of-concept" because it is not just a proof-of-concept now.